src/Processors/QueryPlan/Optimizations/optimizeDirectReadFromTextIndex.cpp

```diff
@@ -182,9 +182,11 @@ class FullTextMatchingFunctionDAGReplacer
         NodesReplacementMap replacements;
         Names original_inputs = actions_dag.getRequiredColumnsNames();
         const auto * filter_node = &actions_dag.findInOutputs(filter_column_name);
+        NamesWithAliases add_aliases;
```

Comment:
wVery Minor: Maybe a name like "required_aliases" or "needed_alias_columns" or "new_aliases" or "added_aliases" may be more descriptive

Reply:
Thanks, I've fixed it.